contextutils/context_and_logging.go

```diff
@@ -42,6 +42,13 @@ const LogLevelEnvName = "LOG_LEVEL"
 func buildProductionLogger() (*zap.Logger, error) {
 	config := zap.NewProductionConfig()
 	config.EncoderConfig.EncodeTime = zapcore.ISO8601TimeEncoder
+
+	// For non-splt logging, allow the user/environment to specify a log file location.
+	// When we log to a file, we will not log it STDOUT.
+	if os.Getenv("LOG_TO_FILE_LOCATION") != "" {
```

Comment:
nit: can we make `LOG_TO_FILE_LOCATION` a constant?

Reply:
good idea, done.